hs_core/tests/api/rest/base.py

```diff
@@ -69,9 +80,8 @@ def getResourceFile(self, res_id, file_name, exhaust_stream=True):
     def _get_file_irods(self, url, exhaust_stream=True):
         response = self.client.get(url, follow=True)
         self.assertTrue(response.status_code, status.HTTP_200_OK)
```

Comment:
While this isn't actually part of your PR, this assertion looks like it was intended to be `assertEqual`.